src/components/material/material-buttons/physical/MaterialButtonReservableFromAnotherLibrary.tsx

```diff
@@ -40,10 +40,8 @@ const MaterialButtonReservableFromAnotherLibrary: FC<
       dataCy={dataCy}
       label={
         size === "small"
-          ? t("reservableFromAnotherLibraryText")
-          : `${t(
-              "reservableFromAnotherLibraryText"
-            )} ${manifestationMaterialType}`
+          ? t("reserveText")
+          : `${t("reserveText")} ${manifestationMaterialType}`
```

Comment:
This is just a note. But it is not the right use of `t`.
`manifestationMaterialType` should have been inserted via a placeholder.